src/global/url.ml

```diff
@@ -46,7 +46,9 @@ let minor v =
 
 let v2 = "https://v2.ocaml.org"
 let manual_with_version v = v2 ^ "/releases/" ^ minor v ^ "/htmlman/index.html"
-let manual = "/releases/latest/manual.html"
+let manual = "/manual"
+let manual_lang v = "/manual/" ^ minor v ^ "/index.html"
+let manual_api v = "/manual/" ^ minor v ^ "/api/index.html"
```

Comment:
`index.html` is the default page looked up by the browser, so we don't need to append this here. Shorter URLs tend to be better. I would rather redirect the other way.

Reply:
The browser doesn't request the default page.  If configured, the server returns a default page when the browser doesn't specify a specific page.

Reply:
When manually testing instructions given in the Dockerfile, I am getting a 404 for /manual/5.1/index.html. What could I be missing?

Reply:
Are your manuals cloned to the top level root, aka /, of your test machine?

Reply:
> Are your manuals cloned to the top level root, aka /, of your test machine?

Ohh! Okay. After copying to /manuals, it works fine. Thanks!

Reply:
This branch: https://github.com/ocaml/ocaml.org/tree/html-compiler-manuals-config contains a patch that allows to set the manual path in a shell environment variable

Reply:
Thanks, I'm on leave this week.  I have cherry-picked your commit to this PR.